### DISCUSSION TITLE
NModal 的 auto-focus 会聚焦到关闭按钮

### DISCUSSION THREAD

**Discussion by ouuan** [tusen-ai/naive-ui#3057] [Ideas]
在关闭按钮可聚焦后，NModal 的 auto-focus 就会自动聚焦到关闭按钮，我感觉这不太好。比较好的应该是，比如说 dialog 预设下优先聚焦到 positive-button，或者是优先聚焦到 NModal 内部的元素而非关闭按钮。如果能由用户指定聚焦到哪或许也很好。

具体实现细节、如何选择应该聚焦到哪个元素我没太想清楚，只是报告一下我发现了这个问题。另外其它组件可能也有类似问题，我不太清楚。

**Comment by 07akioni** [comment]
聚焦到取消按钮可能会好一些，这个未来应该会调整
